kb/temporal-platform-limits-sheet.md

```diff
@@ -37,4 +36,5 @@ Unless otherwise stated, Event limits apply to all Events occurring simultaneous
     - `SignalExternalWorkflowExecution`
```

Comment:
This should be in its own section (concurrent action limit), and not be part of History total count limit.
Also, I would not call them as Events, but rather Actions.